pyodk/_endpoints/submissions.py

```diff
@@ -201,6 +201,8 @@ def create(
         project_id: int | None = None,
         device_id: str | None = None,
         encoding: str = "utf-8",
+        # Here we must use imported typing.List to avoid conflict with .list method
+        attachments: builtins.list[str] | None = None,
```

Comment:
Are you using an old python version? I think the `list` annotation is OK for all current versions. The previous equivalent was to using `typing.List`, not builtins. Anyway that's besides the point - we just need something to feed the `for` loop i.e. `Iterable[PathLike | str]`